components/org.wso2.carbon.identity.remotefetch.core/src/main/java/org/wso2/carbon/identity/remotefetch/core/implementations/configDeployers/VelocityTemplatedSPDeployer.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Update the licese header.

Reply:
[Fixed](https://github.com/wso2-extensions/identity-fetch-remote/pull/12/commits/e33283f45ce3ac37a3c2ef622918b08c838285d4)